<launch>
   <node name="gaze_fixing_node" pkg="manipulation_tutorials" type="gazed_point_fixing_node" respawn="false" output="screen"/>
   <node name="gaze_service_node" pkg="gaze_service" type="gaze_service_test" respawn="false" />

</launch>
